.travis.yml

```diff
@@ -21,6 +21,12 @@ env:
     - TOX_ENV=py34     CONSUL_VERSION=0.6.4
     - TOX_ENV=py35     CONSUL_VERSION=0.6.4
 
+    # consul 0.8.5
```

Comment:
I know that the build matrix is already huge but we skipped 0.7.x release of consul (latest patch version is 0.7.5). If the list is too ling we can always get rid of 0.4.1.